devidda/isef
# Feature/1 setup basic software foundation

Resolves #1 

Initialises a basic foundation we can build on.

How to test:
On PR-branch start a development-server with `yarn dev` and check if you can open the website.
Alternative: `yarn dev --open`
